torch/_inductor/freezing.py

```diff
@@ -40,8 +41,16 @@ def replace_params_with_constants(
         for out_info in fw_metadata.output_info
         if out_info.base_idx is not None
     ]
+
+    # TODO (tmanlaibaatar) figure out why this is different
+    # from mutated_inp_runtime_indices
```

Comment:
cc @bdhirsh

Reply:
oh @tugsbayasgalan - I think `mutated_inp_runtime_indices` **only** refers to mutations that are handled outside the graph (in the AOTAutograd runtime epilogue).

So I think the old logic here was just wrong and happened to work.

This seems ok as a hotfix, but a better fix would probably be to add this metadata directly to `ViewAndMutationMeta` instead of making inductor do even more work here (probably as part of https://github.com/pytorch/pytorch/issues/114403)

Reply:
Yeah i think it might be checking for all inputs maybe instead of all mutated inputs?